cost/azure/unused_sql_databases/azure_unused_sql_databases.pt

```diff
@@ -29,6 +29,14 @@ parameter "param_exclusion_tag_key" do
   description "Azure-native SQL Database tag key to ignore databases. Only supply the tag key. The policy assumes that the tag value is irrelevant."
 end
 
+parameter "param_automatic_delete_db" do
+    type "string"
+    label "Automatically delete unused sql database"
+    description "When this value is set to yes this policy will automatically take the delete unused sql database action"
```

Comment:
Consider:
When this value is set to yes this policy will automatically delete the unused sql database